tmt/steps/execute/scripts/tmt-report-result

```diff
@@ -78,16 +33,16 @@ check_opt_args () {
             OPT="${OPTARG%%=*}"       # extract long option name
         fi
         case "$OPT" in
-            server )         needs_arg; eval "server=\"\$$OPTIND\"" ; OPTIND=$((OPTIND+1)) ;;
-            s )              needs_arg; server=$OPTARG ;;
+            server )         needs_arg; OPTIND=$((OPTIND+1)) ;;
+            s )              needs_arg; ;;
             outputfile )     needs_arg; eval "outputFile=\"\$$OPTIND\"" ; OPTIND=$((OPTIND+1)) ;;
             o )              needs_arg; outputFile="$OPTARG" ;;
-            p )              needs_arg; disablePlugin="$OPTARG" ;;
-            disable-plugin ) needs_arg; eval "disablePlugin=\"\$$OPTIND\"" ; OPTIND=$((OPTIND+1)) ;;
-            t )              needs_arg; message="$OPTARG" ;;
-            message )        needs_arg; eval "message=\"\$$OPTIND\"" ; OPTIND=$((OPTIND+1)) ;;
-            port )           needs_arg; eval "port=\"\$$OPTIND\"" ; OPTIND=$((OPTIND+1)) ;;
-            no-plugins )     disablePlugin=* ;;
+            p )              needs_arg; ;;
+            disable-plugin ) needs_arg; OPTIND=$((OPTIND+1)) ;;
+            t )              needs_arg; ;;
+            message )        needs_arg; OPTIND=$((OPTIND+1)) ;;
+            port )           needs_arg; OPTIND=$((OPTIND+1)) ;;
+            no-plugins )     ;;
```

Comment:
Perhaps the help message should be updated as well and state that these options are ignored?

Reply:
The help message was updated, though I'm not sure about help text as a whole. @happz @thrix Could you take a look please?

Reply:
Correct, LGTM.